packages/shared/components/NetworkIconBadge.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="typescript">
+    import { NetworkIcon, Text, Tooltip } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { NetworkProtocol, NetworkType } from '@core/network'
+
+    let tooltipAnchor
+    let showTooltip = false
+
+    export let networkType: NetworkType
```

Comment:
Stick the exports directly underneath the imports so that we can easily find them and each file is consistent